xrayvision/tests/test_imaging.py

```diff
@@ -208,22 +219,23 @@ def test_vis_to_map(m, n, pos, pixel):
 
     vis = map_to_vis(mp, u=u, v=v)
 
-    res = vis_to_map(vis, shape=(m, n) * apu.pixel, pixel_size=pixel, natural=False)
-    # assert np.allclose(res.data, data)
+    res = vis_to_map(vis, shape=(m, n) * apu.pixel, pixel_size=pixel, scheme="natural")
+    # TODO: figure out why atol is needed here
+    assert_allclose(data, res.data, atol=1e-15)
 
-    assert res.reference_coordinate.Tx == pos[0]
-    assert res.reference_coordinate.Ty == pos[1]
-    assert res.scale.axis1 == pixel[0] / apu.pix
-    assert res.scale.axis2 == pixel[1] / apu.pix
+    assert res.reference_coordinate.Tx == pos[1]
+    assert res.reference_coordinate.Ty == pos[0]
+    assert res.scale.axis1 == pixel[1]
+    assert res.scale.axis2 == pixel[0]
```

Comment:
Why does the x-axis correspond to index 1 and the y-axis corresponds to index 2 for both the pixel and the pos arrays? The same comment applies to lines 210-213

Reply:
This is due to WCS which is (originally written in FORTRAN) is column-major vs python which is row-major. In the this specific case `Tx` is associated with `ctype1 `cdelt1` etc which corresponds to python array index 1 and `Ty` with `ctype2`, `cdelt2` to python array in python array index 0

Reply:
Ok, thanks!